app/views/new_recipes/show.html.erb

```diff
@@ -27,7 +27,10 @@
           <% if @recipe.user_id == current_user.id %>
             <div class="flex-container my-5 pb-4">
               <%= link_to 'Generate shopping list', shopping_list_path, class: "btn btn-info" %>
-              <%= render 'ingredient' %>
+              <% if current_user.id == @recipe.user_id %>
+              <%= link_to 'Add Ingredient', new_user_new_recipe_recipe_food_path(current_user, @recipe), class: "btn btn-info" %>
+            <% end %>
+            
             </div>
           <% end %>
         </div>
```

Comment:
![image](https://github.com/YoutLone/Recipe/assets/93039359/6e85b522-e4d4-4e9e-b226-42b305859e34)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on adding the recipe but I faced a problem that we cannot make a recipe public once it is created private or vice versa :fearful: Kindly fix this issue to make your app work as project requirements :+1: